src/java/org/apache/cassandra/security/ThreadAwareSecurityManager.java

```diff
@@ -208,7 +225,5 @@ public void checkPackageAccess(String pkg)
             RuntimePermission perm = new RuntimePermission("accessClassInPackage." + pkg);
             throw new AccessControlException("access denied: " + perm, perm);
         }
-
-        super.checkPackageAccess(pkg);
```

Comment:
Otherwise JavaScript based UDFs stop working due to JPMS, that would raise errors like
`Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "accessClassInPackage.jdk.internal.org.objectweb.asm")`